### PR TITLE
Add agrvate to Bioconda recipes

### PR DESCRIPTION
**Pull request by VishnuRaghuram94**
AgrVATE is a tool for rapid identification of Staphylococcus aureus agr locus type and also reports possible variants in the agr operon.

Source: https://github.com/VishnuRaghuram94/AgrVATE

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

File: recipes/agrvate/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+source:
+  url: https://github.com/VishnuRaghuram94/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 08126f72bfb4bb3a177168f7df06104d2c39177ae72cf65dd10e86ab7e94b50a
+
+requirements:
+  run:
+    - blast
+    - hmmer
+    - seqkit
```
Comment (fanninpm) [inline recipes/agrvate/meta.yaml:19]:
```suggestion
    - seqkit
```

File: recipes/agrvate/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+{% set version = "1.0" %}
+
+package:
+  name: AgrVATE
```
Comment (fanninpm) [inline recipes/agrvate/meta.yaml:4]:
```suggestion
{% set name = "AgrVATE" %}
{% set version = "1.0" %}

package:
  name: {{ name|lower }}
```

**Comment by VishnuRaghuram94** [comment]
Local test was successful :

	15:28:15 > conda create -n agrvate -c conda-forge -c bioconda -c https://151866-42372094-gh.circle-artifacts.com/0/tmp/artifacts/packages agrvate
	15:28:29 > conda activate agrvate

	(agrvate) 15:28:47 > agrvate 

	AgrVATE: Agr Variant Assessment & Typing Engine

	VERSION: agrvate v1.0

	USAGE:   agrvate [options] -i filename.fasta

	FLAGS:
	  -i | --input           Input S. aureus genome in FASTA format
	  -t | --typing-only     Does agr typing only (skips agr operon extraction and frameshift detection)
	  -m | --mummer          Uses mummer instead of usearch (May not perform frameshift detection)
	  -f | --force           Force overwrite existing results directory
	  -d | --databases       Path to agrvate_databases (Not required if installed using Conda)
	  -h | --help            Print this help message and exit
	  -v | --version         Print version and exit

	SOURCE:  https://github.com/VishnuRaghuram94/AgrVATE

	(agrvate) 15:29:57 > agrvate --version
	agrvate v1.0

	(agrvate) 15:38:12 > agrvate -i test_input/USA300.fna
	Processing test_input/USA300.fna .../home/vraghu2/miniconda3/envs/agrvate/bin/agrvate_databases/ is valid
	agr typing successful, gp1
	usearch found
	agr operon extraction successful
	Snippy successful
	No frameshifts found 

	(agrvate) 15:38:21 > ls USA300-results/
	USA300-agr_gp.tab                  USA300-blastn_log.txt  USA300-snippy-log.txt
	USA300-agr_operon.fna              USA300-pcr-log.txt     USA300-summary.tab   
	USA300-agr_operon_frameshifts.tab  USA300-snippy

**Comment by fanninpm** [comment]
@BiocondaBot please add label

**Comment by VishnuRaghuram94** [comment]
@BiocondaBot please add label

**Review by rpetit3 — APPROVED** [review]

**Comment by rpetit3** [comment]
@biocondabot please merge

**Pull request merged**